rest_framework_reactive/pool.py

```diff
@@ -265,7 +272,7 @@ def stop_all(self):
         Stops all query observers.
         """
 
-        for observer in self._observers.values():
+        for observer in list(self._observers.values()):
```

Comment:
It can't because it modifies the underlying data structure so a copy should be made. In Python 2 a copy is made, while in Python 3 you get a view, but you really need a copy here.